backend/src/main/resources/application-dev.yaml

```diff
@@ -22,3 +22,5 @@ simple-report:
   data-hub:
     upload-enabled: ${DATAHUB_UPLOAD_ENABLE:false}
     upload-url: "https://prime-data-hub-test.azurefd.net/api/reports?option=SkipSend" # ValidatePayload, SkipSend, SkipInvalidItems
+twilio:
+  enabled: false
```

Comment:
Why is this false here but true in `application-azure-dev.yaml`

Reply:
Most developers won't have Twilio creds in their shell environment, where the `DEV` profile is active, but `AZURE-DEV` isn't – and we _do_ want Twilio enabled in the deployed dev env